teamcity: failed tests on release-2.1: testrace/TestAuthenticationAPIUserLogin, test/TestAuthenticationAPIUserLogin

The following tests appear to have failed:

[#900711](https://teamcity.cockroachdb.com/viewLog.html?buildId=900711):


```
--- FAIL: testrace/TestAuthenticationAPIUserLogin (38.470s)
authentication_test.go:448: login got error Post https://127.0.0.1:34827/login: net/http: request canceled (Client.Timeout exceeded while awaiting headers), wanted error with 401 status
------- Stdout: -------
W180912 20:10:17.995789 18389 server/status/runtime.go:310  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180912 20:10:18.370744 18389 server/server.go:849  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180912 20:10:18.372632 18389 base/addr_validation.go:279  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180912 20:10:18.372817 18389 base/addr_validation.go:319  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180912 20:10:18.427602 18389 server/config.go:493  [n?] 1 storage engine initialized
I180912 20:10:18.427762 18389 server/config.go:496  [n?] RocksDB cache size: 128 MiB
I180912 20:10:18.427807 18389 server/config.go:496  [n?] store 0: in-memory, size 0 B
I180912 20:10:18.809647 18389 server/node.go:373  [n?] **** cluster a9c9ae71-d681-44cf-8706-59cff4642945 has been created
I180912 20:10:18.809820 18389 server/server.go:1411  [n?] **** add additional nodes by specifying --join=127.0.0.1:43671
I180912 20:10:18.815444 18389 gossip/gossip.go:407  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:43671" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:12 > build_tag:"v2.1.0-beta.20180904-436-g7075213" started_at:1536783018813956842 
I180912 20:10:18.821263 18457 gossip/gossip.go:941  [n1] gossip connectivity
 
I180912 20:10:18.859829 18389 storage/store.go:1551  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180912 20:10:18.861228 18389 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180912 20:10:18.861646 18389 storage/stores.go:242  [n1] read 0 node addresses from persistent storage
I180912 20:10:18.862205 18389 server/node.go:699  [n1] connecting to gossip network to verify cluster ID...
I180912 20:10:18.862405 18389 server/node.go:724  [n1] node connected via gossip and verified as part of cluster "a9c9ae71-d681-44cf-8706-59cff4642945"
I180912 20:10:18.862795 18389 server/node.go:548  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180912 20:10:18.864011 18389 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:18.864090 18389 server/server.go:1802  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180912 20:10:18.866161 18389 server/server.go:1548  [n1] starting https server at 127.0.0.1:34827 (use: 127.0.0.1:34827)
I180912 20:10:18.866294 18389 server/server.go:1550  [n1] starting grpc/postgres server at 127.0.0.1:43671
I180912 20:10:18.866355 18389 server/server.go:1551  [n1] advertising CockroachDB node at 127.0.0.1:43671
W180912 20:10:18.866699 18389 jobs/registry.go:309  [n1] unable to get node liveness: node not in the liveness table
I180912 20:10:18.889329 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:19.273745 18536 storage/replica_command.go:298  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
W180912 20:10:20.138234 18552 storage/store.go:3936  [n1,s1] handle raft ready: 0.5s [processed=2]
I180912 20:10:20.454319 18457 gossip/gossip.go:941  [n1] gossip connectivity
  n1 [sentinel];
 
I180912 20:10:20.736662 18675 storage/replica_command.go:298  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
W180912 20:10:20.833269 18463 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "unnamed" id=8f5233c9 key=/Table/SystemConfigSpan/Start rw=true pri=0.02746657 iso=SERIALIZABLE stat=PENDING epo=0 ts=1536783019.296046008,0 orig=1536783019.296046008,0 max=1536783019.296046008,0 wto=false rop=false seq=6
I180912 20:10:21.128505 18706 storage/replica_command.go:298  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180912 20:10:21.285431 18667 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180912 20:10:21.370942 18669 rpc/nodedialer/nodedialer.go:92  [consistencyChecker,n1,s1,r1/1:/{Min-System/}] connection to n1 established
I180912 20:10:21.851995 18430 storage/replica_command.go:298  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180912 20:10:22.265389 18725 storage/replica_command.go:298  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180912 20:10:22.913337 18707 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:2.0-12 User:root}
I180912 20:10:23.046398 18727 storage/replica_command.go:298  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180912 20:10:23.375335 18431 storage/replica_command.go:298  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180912 20:10:23.685181 18678 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180912 20:10:24.112110 18433 storage/replica_command.go:298  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180912 20:10:24.513326 18787 storage/replica_command.go:298  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180912 20:10:24.684011 18743 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:a9c6dc4d-210a-488e-8d47-2e1bfb78313a User:root}
I180912 20:10:24.822676 18758 storage/replica_command.go:298  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180912 20:10:25.187949 18773 storage/replica_command.go:298  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180912 20:10:25.332938 18761 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180912 20:10:25.556642 18834 storage/replica_command.go:298  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180912 20:10:25.579106 18713 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180912 20:10:25.662051 18389 server/server.go:1604  [n1] done ensuring all necessary migrations have run
I180912 20:10:25.662361 18389 server/server.go:1607  [n1] serving sql connections
I180912 20:10:25.832632 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.833916 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.834534 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.835115 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.835677 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.836271 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.837405 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.838068 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.838583 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.839075 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.839593 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.840092 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.840552 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.841059 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.841609 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.842162 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.842924 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.843713 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.844542 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.845522 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.846731 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.848750 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.852238 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.867275 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.884348 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.895212 18767 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180912 20:10:25.967868 18820 storage/replica_command.go:298  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180912 20:10:25.968973 18769 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:43671} Attrs: Locality: ServerVersion:2.0-12 BuildTag:v2.1.0-beta.20180904-436-g7075213 StartedAt:1536783018813956842 LocalityAddress:[]} ClusterID:a9c9ae71-d681-44cf-8706-59cff4642945 StartedAt:1536783018813956842 LastUp:1536783018813956842}
I180912 20:10:26.056295 18389 server/testserver.go:420  had 13 ranges at startup, expected 22
I180912 20:10:26.092940 18389 server/testserver.go:420  had 13 ranges at startup, expected 22
I180912 20:10:26.161460 18389 server/testserver.go:420  had 13 ranges at startup, expected 22
I180912 20:10:26.332745 18823 storage/replica_command.go:298  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180912 20:10:26.376600 18389 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 20:10:26.646677 18389 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 20:10:26.832962 18883 storage/replica_command.go:298  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180912 20:10:27.169453 18886 storage/replica_command.go:298  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180912 20:10:27.189525 18389 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 20:10:27.428544 18595 storage/replica_proposal.go:211  [n1,s1,r10/1:/Table/1{3-4}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.422676650,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.442447 18594 storage/replica_proposal.go:211  [n1,s1,r16/1:/Table/{19-20}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.438950450,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.459673 18590 storage/replica_proposal.go:211  [n1,s1,r17/1:/{Table/20-Max}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.374134997,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.476622 18898 storage/replica_command.go:298  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180912 20:10:27.542475 18597 storage/replica_proposal.go:211  [n1,s1,r7/1:/Table/{SystemCon…-11}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.538640959,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.760501 18893 storage/replica_command.go:298  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180912 20:10:27.997538 18897 storage/replica_command.go:298  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180912 20:10:28.182311 18908 storage/replica_command.go:298  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180912 20:10:28.280378 18389 server/testserver.go:420  had 20 ranges at startup, expected 22
I180912 20:10:28.409982 18877 storage/replica_command.go:298  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180912 20:10:28.896996 18644 server/status/runtime.go:484  [n1] runtime stats: 540 MiB RSS, 236 goroutines, 0 B/0 B/0 B GO alloc/idle/total, 37 MiB/81 MiB CGO alloc/total, 0.0 CGO/sec, 0.0/0.0 %(u/s)time, 0.0 %gc (222x), 0.0/0.0/0 (%r/%w/q)disk, 940 KiB/940 KiB (r/w)net
I180912 20:10:28.976967 18606 storage/replica_proposal.go:211  [n1,s1,r13/1:/Table/1{6-7}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783028.973154655,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:29.292087 18547 storage/replica_proposal.go:211  [n1,s1,r4/1:/System/{NodeLive…-tsd}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783029.287031412,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:29.491182 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:30.980563 18546 storage/replica_proposal.go:211  [n1,s1,r11/1:/Table/1{4-5}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783030.976847244,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:31.983123 18562 storage/replica_proposal.go:211  [n1,s1,r9/1:/Table/1{2-3}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783031.979068640,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:32.143827 18566 storage/replica_proposal.go:211  [n1,s1,r5/1:/System/ts{d-e}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783032.094141131,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:34.988287 18578 storage/replica_proposal.go:211  [n1,s1,r12/1:/Table/1{5-6}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783034.984419367,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:37.999029 18588 storage/replica_proposal.go:211  [n1,s1,r15/1:/Table/1{8-9}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783037.989417870,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:38.899707 18644 server/status/runtime.go:484  [n1] runtime stats: 541 MiB RSS, 239 goroutines, 15 MiB/36 MiB/66 MiB GO alloc/idle/total, 38 MiB/82 MiB CGO alloc/total, 79.7 CGO/sec, 25.8/0.3 %(u/s)time, 1.2 %gc (1x), 1.4/1.9/0 (%r/%w/q)disk, 966 KiB/966 KiB (r/w)net
I180912 20:10:38.995797 18586 storage/replica_proposal.go:211  [n1,s1,r6/1:/{System/tse-Table/System…}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783038.991098391,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:39.491085 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:45.104290 18548 storage/replica_proposal.go:211  [n1,s1,r14/1:/Table/1{7-8}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783045.098755949,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:46.088974 18557 storage/replica_proposal.go:211  [n1,s1,r8/1:/Table/1{1-2}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783046.083804813,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:48.902315 18644 server/status/runtime.go:484  [n1] runtime stats: 543 MiB RSS, 246 goroutines, 22 MiB/29 MiB/66 MiB GO alloc/idle/total, 38 MiB/82 MiB CGO alloc/total, 86.7 CGO/sec, 23.0/0.3 %(u/s)time, 2.7 %gc (2x), 9.4/1.6/0 (%r/%w/q)disk, 1.5 MiB/1.5 MiB (r/w)net
I180912 20:10:49.491081 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:54.944231 18389 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
1      [async] closedts-rangefeed-subscriber
I180912 20:10:54.967542 18389 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
I180912 20:10:55.996549 18665 kv/transport_race.go:91  transport race promotion: ran 49 iterations on up to 785 requests
--- FAIL: test/TestAuthenticationAPIUserLogin (39.490s)

------- Stdout: -------
W180912 19:56:48.191129 18566 server/status/runtime.go:310  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180912 19:56:48.198578 18566 server/server.go:849  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180912 19:56:48.198715 18566 base/addr_validation.go:279  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180912 19:56:48.198731 18566 base/addr_validation.go:319  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180912 19:56:48.201208 18566 server/config.go:493  [n?] 1 storage engine initialized
I180912 19:56:48.201249 18566 server/config.go:496  [n?] RocksDB cache size: 128 MiB
I180912 19:56:48.201263 18566 server/config.go:496  [n?] store 0: in-memory, size 0 B
I180912 19:56:48.204345 18566 server/node.go:373  [n?] **** cluster 227671eb-3c30-4101-a572-7cdc3731529a has been created
I180912 19:56:48.204371 18566 server/server.go:1411  [n?] **** add additional nodes by specifying --join=127.0.0.1:38979
I180912 19:56:48.204494 18566 gossip/gossip.go:407  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:38979" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:12 > build_tag:"v2.1.0-beta.20180904-436-g7075213" started_at:1536782208204432430 
I180912 19:56:48.204742 18622 gossip/gossip.go:941  [n1] gossip connectivity
 
I180912 19:56:48.205410 18566 storage/store.go:1551  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180912 19:56:48.205497 18566 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180912 19:56:48.205540 18566 storage/stores.go:242  [n1] read 0 node addresses from persistent storage
I180912 19:56:48.205599 18566 server/node.go:699  [n1] connecting to gossip network to verify cluster ID...
I180912 19:56:48.205621 18566 server/node.go:724  [n1] node connected via gossip and verified as part of cluster "227671eb-3c30-4101-a572-7cdc3731529a"
I180912 19:56:48.205643 18566 server/node.go:548  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180912 19:56:48.206976 18566 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 19:56:48.207005 18566 server/server.go:1802  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180912 19:56:48.207065 18566 server/server.go:1548  [n1] starting https server at 127.0.0.1:44823 (use: 127.0.0.1:44823)
I180912 19:56:48.207085 18566 server/server.go:1550  [n1] starting grpc/postgres server at 127.0.0.1:38979
I180912 19:56:48.207104 18566 server/server.go:1551  [n1] advertising CockroachDB node at 127.0.0.1:38979
I180912 19:56:48.212887 18639 storage/replica_command.go:298  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180912 19:56:48.221319 18799 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 19:56:48.606775 18805 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180912 19:56:48.610950 18544 storage/replica_command.go:298  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180912 19:56:48.617207 18834 storage/replica_command.go:298  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180912 19:56:48.626629 18808 storage/replica_command.go:298  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180912 19:56:48.635421 18851 storage/replica_command.go:298  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180912 19:56:48.644662 18829 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:2.0-12 User:root}
I180912 19:56:48.648953 18044 storage/replica_command.go:298  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180912 19:56:48.660546 18868 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180912 19:56:48.662096 18590 storage/replica_command.go:298  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180912 19:56:48.676744 18049 storage/replica_command.go:298  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180912 19:56:48.680901 18914 storage/replica_command.go:298  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180912 19:56:48.686888 18916 storage/replica_command.go:298  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180912 19:56:48.690623 18890 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:0a4e5566-2a1f-49e0-9ad2-dad49d9c6f84 User:root}
I180912 19:56:48.692106 18925 storage/replica_command.go:298  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180912 19:56:48.701471 18897 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180912 19:56:48.701964 18907 storage/replica_command.go:298  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180912 19:56:48.707834 18913 storage/replica_command.go:298  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180912 19:56:48.712519 19004 storage/replica_command.go:298  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180912 19:56:48.714273 18936 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180912 19:56:48.716662 18566 server/server.go:1604  [n1] done ensuring all necessary migrations have run
I180912 19:56:48.716706 18566 server/server.go:1607  [n1] serving sql connections
I180912 19:56:48.717044 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.720291 18968 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180912 19:56:48.722661 18970 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:38979} Attrs: Locality: ServerVersion:2.0-12 BuildTag:v2.1.0-beta.20180904-436-g7075213 StartedAt:1536782208204432430 LocalityAddress:[]} ClusterID:227671eb-3c30-4101-a572-7cdc3731529a StartedAt:1536782208204432430 LastUp:1536782208204432430}
I180912 19:56:48.724578 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.724744 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.724874 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.725049 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.725707 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.725852 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.726802 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.726927 18566 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 19:56:48.730224 18566 server/testserver.go:420  had 15 ranges at startup, expected 22
I180912 19:56:48.731505 19014 storage/replica_command.go:298  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180912 19:56:48.734463 18566 server/testserver.go:420  had 15 ranges at startup, expected 22
I180912 19:56:48.735105 18566 server/testserver.go:420  had 15 ranges at startup, expected 22
W180912 19:56:48.748534 19029 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "split" id=9dce8c65 key=/Local/Range/Table/18/RangeDescriptor rw=true pri=0.00817426 iso=SERIALIZABLE stat=PENDING epo=0 ts=1536782208.734560257,1 orig=1536782208.731534196,0 max=1536782208.731534196,0 wto=false rop=false seq=1
I180912 19:56:48.753527 18990 storage/replica_command.go:298  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180912 19:56:48.754520 18566 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 19:56:48.755070 18566 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 19:56:48.755512 18566 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 19:56:48.755826 18566 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 19:56:48.756073 18566 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 19:56:48.757134 18566 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 19:56:48.760477 19040 storage/replica_command.go:298  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180912 19:56:48.764696 18948 storage/replica_command.go:298  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180912 19:56:48.765116 18566 server/testserver.go:420  had 18 ranges at startup, expected 22
I180912 19:56:48.765978 18566 server/testserver.go:420  had 18 ranges at startup, expected 22
I180912 19:56:48.767174 18566 server/testserver.go:420  had 18 ranges at startup, expected 22
I180912 19:56:48.770145 18566 server/testserver.go:420  had 19 ranges at startup, expected 22
I180912 19:56:48.770284 18954 storage/replica_command.go:298  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180912 19:56:48.772647 18566 server/testserver.go:420  had 19 ranges at startup, expected 22
I180912 19:56:48.774936 18958 storage/replica_command.go:298  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180912 19:56:48.777239 18566 server/testserver.go:420  had 20 ranges at startup, expected 22
I180912 19:56:48.779311 18872 storage/replica_command.go:298  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180912 19:56:49.170920 18566 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] closedts-rangefeed-subscriberauthentication_test.go:448: login got error Post https://127.0.0.1:34827/login: net/http: request canceled (Client.Timeout exceeded while awaiting headers), wanted error with 401 status
------- Stdout: -------
W180912 20:10:17.995789 18389 server/status/runtime.go:310  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180912 20:10:18.370744 18389 server/server.go:849  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180912 20:10:18.372632 18389 base/addr_validation.go:279  [n?] server certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180912 20:10:18.372817 18389 base/addr_validation.go:319  [n?] web UI certificate addresses: IP=127.0.0.1,::1; DNS=localhost,*.local; CN=node
I180912 20:10:18.427602 18389 server/config.go:493  [n?] 1 storage engine initialized
I180912 20:10:18.427762 18389 server/config.go:496  [n?] RocksDB cache size: 128 MiB
I180912 20:10:18.427807 18389 server/config.go:496  [n?] store 0: in-memory, size 0 B
I180912 20:10:18.809647 18389 server/node.go:373  [n?] **** cluster a9c9ae71-d681-44cf-8706-59cff4642945 has been created
I180912 20:10:18.809820 18389 server/server.go:1411  [n?] **** add additional nodes by specifying --join=127.0.0.1:43671
I180912 20:10:18.815444 18389 gossip/gossip.go:407  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:43671" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:12 > build_tag:"v2.1.0-beta.20180904-436-g7075213" started_at:1536783018813956842 
I180912 20:10:18.821263 18457 gossip/gossip.go:941  [n1] gossip connectivity
 
I180912 20:10:18.859829 18389 storage/store.go:1551  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180912 20:10:18.861228 18389 server/node.go:476  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7103.00 p25=7103.00 p50=7103.00 p75=7103.00 p90=7103.00 pMax=7103.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180912 20:10:18.861646 18389 storage/stores.go:242  [n1] read 0 node addresses from persistent storage
I180912 20:10:18.862205 18389 server/node.go:699  [n1] connecting to gossip network to verify cluster ID...
I180912 20:10:18.862405 18389 server/node.go:724  [n1] node connected via gossip and verified as part of cluster "a9c9ae71-d681-44cf-8706-59cff4642945"
I180912 20:10:18.862795 18389 server/node.go:548  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180912 20:10:18.864011 18389 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:18.864090 18389 server/server.go:1802  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180912 20:10:18.866161 18389 server/server.go:1548  [n1] starting https server at 127.0.0.1:34827 (use: 127.0.0.1:34827)
I180912 20:10:18.866294 18389 server/server.go:1550  [n1] starting grpc/postgres server at 127.0.0.1:43671
I180912 20:10:18.866355 18389 server/server.go:1551  [n1] advertising CockroachDB node at 127.0.0.1:43671
W180912 20:10:18.866699 18389 jobs/registry.go:309  [n1] unable to get node liveness: node not in the liveness table
I180912 20:10:18.889329 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:19.273745 18536 storage/replica_command.go:298  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
W180912 20:10:20.138234 18552 storage/store.go:3936  [n1,s1] handle raft ready: 0.5s [processed=2]
I180912 20:10:20.454319 18457 gossip/gossip.go:941  [n1] gossip connectivity
  n1 [sentinel];
 
I180912 20:10:20.736662 18675 storage/replica_command.go:298  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
W180912 20:10:20.833269 18463 storage/intent_resolver.go:668  [n1,s1] failed to push during intent resolution: failed to push "unnamed" id=8f5233c9 key=/Table/SystemConfigSpan/Start rw=true pri=0.02746657 iso=SERIALIZABLE stat=PENDING epo=0 ts=1536783019.296046008,0 orig=1536783019.296046008,0 max=1536783019.296046008,0 wto=false rop=false seq=6
I180912 20:10:21.128505 18706 storage/replica_command.go:298  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180912 20:10:21.285431 18667 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180912 20:10:21.370942 18669 rpc/nodedialer/nodedialer.go:92  [consistencyChecker,n1,s1,r1/1:/{Min-System/}] connection to n1 established
I180912 20:10:21.851995 18430 storage/replica_command.go:298  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180912 20:10:22.265389 18725 storage/replica_command.go:298  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180912 20:10:22.913337 18707 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:2.0-12 User:root}
I180912 20:10:23.046398 18727 storage/replica_command.go:298  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180912 20:10:23.375335 18431 storage/replica_command.go:298  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
I180912 20:10:23.685181 18678 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180912 20:10:24.112110 18433 storage/replica_command.go:298  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180912 20:10:24.513326 18787 storage/replica_command.go:298  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180912 20:10:24.684011 18743 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:a9c6dc4d-210a-488e-8d47-2e1bfb78313a User:root}
I180912 20:10:24.822676 18758 storage/replica_command.go:298  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180912 20:10:25.187949 18773 storage/replica_command.go:298  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180912 20:10:25.332938 18761 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180912 20:10:25.556642 18834 storage/replica_command.go:298  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180912 20:10:25.579106 18713 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180912 20:10:25.662051 18389 server/server.go:1604  [n1] done ensuring all necessary migrations have run
I180912 20:10:25.662361 18389 server/server.go:1607  [n1] serving sql connections
I180912 20:10:25.832632 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.833916 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.834534 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.835115 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.835677 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.836271 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.837405 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.838068 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.838583 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.839075 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.839593 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.840092 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.840552 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.841059 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.841609 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.842162 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.842924 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.843713 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.844542 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.845522 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.846731 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.848750 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.852238 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.867275 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.884348 18389 server/testserver.go:420  had 12 ranges at startup, expected 22
I180912 20:10:25.895212 18767 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180912 20:10:25.967868 18820 storage/replica_command.go:298  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180912 20:10:25.968973 18769 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:43671} Attrs: Locality: ServerVersion:2.0-12 BuildTag:v2.1.0-beta.20180904-436-g7075213 StartedAt:1536783018813956842 LocalityAddress:[]} ClusterID:a9c9ae71-d681-44cf-8706-59cff4642945 StartedAt:1536783018813956842 LastUp:1536783018813956842}
I180912 20:10:26.056295 18389 server/testserver.go:420  had 13 ranges at startup, expected 22
I180912 20:10:26.092940 18389 server/testserver.go:420  had 13 ranges at startup, expected 22
I180912 20:10:26.161460 18389 server/testserver.go:420  had 13 ranges at startup, expected 22
I180912 20:10:26.332745 18823 storage/replica_command.go:298  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180912 20:10:26.376600 18389 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 20:10:26.646677 18389 server/testserver.go:420  had 14 ranges at startup, expected 22
I180912 20:10:26.832962 18883 storage/replica_command.go:298  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180912 20:10:27.169453 18886 storage/replica_command.go:298  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
I180912 20:10:27.189525 18389 server/testserver.go:420  had 16 ranges at startup, expected 22
I180912 20:10:27.428544 18595 storage/replica_proposal.go:211  [n1,s1,r10/1:/Table/1{3-4}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.422676650,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.442447 18594 storage/replica_proposal.go:211  [n1,s1,r16/1:/Table/{19-20}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.438950450,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.459673 18590 storage/replica_proposal.go:211  [n1,s1,r17/1:/{Table/20-Max}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.374134997,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.476622 18898 storage/replica_command.go:298  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180912 20:10:27.542475 18597 storage/replica_proposal.go:211  [n1,s1,r7/1:/Table/{SystemCon…-11}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783027.538640959,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:27.760501 18893 storage/replica_command.go:298  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180912 20:10:27.997538 18897 storage/replica_command.go:298  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180912 20:10:28.182311 18908 storage/replica_command.go:298  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180912 20:10:28.280378 18389 server/testserver.go:420  had 20 ranges at startup, expected 22
I180912 20:10:28.409982 18877 storage/replica_command.go:298  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180912 20:10:28.896996 18644 server/status/runtime.go:484  [n1] runtime stats: 540 MiB RSS, 236 goroutines, 0 B/0 B/0 B GO alloc/idle/total, 37 MiB/81 MiB CGO alloc/total, 0.0 CGO/sec, 0.0/0.0 %(u/s)time, 0.0 %gc (222x), 0.0/0.0/0 (%r/%w/q)disk, 940 KiB/940 KiB (r/w)net
I180912 20:10:28.976967 18606 storage/replica_proposal.go:211  [n1,s1,r13/1:/Table/1{6-7}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783028.973154655,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:29.292087 18547 storage/replica_proposal.go:211  [n1,s1,r4/1:/System/{NodeLive…-tsd}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783029.287031412,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:29.491182 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:30.980563 18546 storage/replica_proposal.go:211  [n1,s1,r11/1:/Table/1{4-5}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783030.976847244,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:31.983123 18562 storage/replica_proposal.go:211  [n1,s1,r9/1:/Table/1{2-3}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783031.979068640,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:32.143827 18566 storage/replica_proposal.go:211  [n1,s1,r5/1:/System/ts{d-e}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783032.094141131,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:34.988287 18578 storage/replica_proposal.go:211  [n1,s1,r12/1:/Table/1{5-6}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783034.984419367,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:37.999029 18588 storage/replica_proposal.go:211  [n1,s1,r15/1:/Table/1{8-9}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783037.989417870,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:38.899707 18644 server/status/runtime.go:484  [n1] runtime stats: 541 MiB RSS, 239 goroutines, 15 MiB/36 MiB/66 MiB GO alloc/idle/total, 38 MiB/82 MiB CGO alloc/total, 79.7 CGO/sec, 25.8/0.3 %(u/s)time, 1.2 %gc (1x), 1.4/1.9/0 (%r/%w/q)disk, 966 KiB/966 KiB (r/w)net
I180912 20:10:38.995797 18586 storage/replica_proposal.go:211  [n1,s1,r6/1:/{System/tse-Table/System…}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783038.991098391,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:39.491085 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:45.104290 18548 storage/replica_proposal.go:211  [n1,s1,r14/1:/Table/1{7-8}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783045.098755949,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:46.088974 18557 storage/replica_proposal.go:211  [n1,s1,r8/1:/Table/1{1-2}] new range lease repl=(n1,s1):1 seq=3 start=1536783018.838447052,0 epo=1 pro=1536783046.083804813,0 following repl=(n1,s1):1 seq=2 start=1536783018.838447052,0 exp=1536783027.854903656,0 pro=1536783018.854963128,0
I180912 20:10:48.902315 18644 server/status/runtime.go:484  [n1] runtime stats: 543 MiB RSS, 246 goroutines, 22 MiB/29 MiB/66 MiB GO alloc/idle/total, 38 MiB/82 MiB CGO alloc/total, 86.7 CGO/sec, 23.0/0.3 %(u/s)time, 2.7 %gc (2x), 9.4/1.6/0 (%r/%w/q)disk, 1.5 MiB/1.5 MiB (r/w)net
I180912 20:10:49.491081 18646 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180912 20:10:54.944231 18389 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
1      [async] closedts-rangefeed-subscriber
I180912 20:10:54.967542 18389 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
I180912 20:10:55.996549 18665 kv/transport_race.go:91  transport race promotion: ran 49 iterations on up to 785 requests

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/server/authentication_test.go
+++ b/pkg/server/authentication_test.go
@@ -428,6 +428,9 @@ func TestAuthenticationAPIUserLogin(t *testing.T) {
 		// We need to instantiate our own HTTP Request, because we must inspect
 		// the returned headers.
 		httpClient, err := ts.GetHTTPClient()
+		if util.RaceEnabled {
+			httpClient.Timeout += 30 * time.Second
+		}
 		if err != nil {
 			t.Fatalf("could not get HTTP client: %s", err)
 		}

